modulefiles/build.ursa.gnu.lua

```diff
@@ -0,0 +1,60 @@
+help([[
+Load environment to compile UFS_UTILS on Ursa using Gnu
+]])
+
+prepend_path("MODULEPATH", "/contrib/spack-stack/spack-stack-1.9.1/envs/ue-gcc-12.4.0/install/modulefiles/Core")
```

Comment:
Noting, this might be 1.9.2 by the time the UFSWM Ursa port is completed.

Reply:
The UFSWM timeline is spack-stack 1.9.1 PR then Ursa port PR. I think 1.9.2 will be installed within the week on machines, but i don't know what they wanted to do with the UFSWM SS-191 PR.

Reply:
Done at c254bd8.